py/desispec/io/fibermap.py

```diff
@@ -590,20 +591,16 @@ def assemble_fibermap(night, expid, badamps=None, badfibers_filename=None,
         log.error('Using fiberassign from an earlier exposure %08d/%08d', night, rawfafile_expid)
 
     #- Look for override fiberassign file in svn
-    if allow_svn_override and ('DESI_TARGET' in os.environ):
-        targdir = os.getenv('DESI_TARGET')
-        testfile = f'{targdir}/fiberassign/tiles/trunk/{tileid//1000:03d}/fiberassign-{tileid:06d}.fits'
-        try:
-            fafile = checkgzip(testfile)
-        except FileNotFoundError:
-            # no alternate fiberassign file in svn yet, that's ok
-            pass
-
-        if rawfafile != fafile:
-            log.info(f'Overriding raw fiberassign file {rawfafile} with svn {fafile}')
+    if allow_svn_override:
+        if 'DESI_TILES' in os.environ:
```

Comment:
Technically, this violates our proposed rule of 'if `DESI_TILES` is not defined, fail', so I can make this stricter if necessary.